src/alfasim_sdk/_internal/constants.py

```diff
@@ -234,6 +234,8 @@ class MaterialType(Enum):
 FLUID_GAS = "gas"
 FLUID_OIL = "oil"
 FLUID_WATER = "water"
+FLUID_DROPLET = "droplet"
+FLUID_BUBBLE = "bubble"
```

Comment:
Are you sure this is required? Note that below we have `FLUID_PHASE_NAMES = [FLUID_GAS, FLUID_OIL, FLUID_WATER]`

It seems that `FLUID_DROPLET` and `FLUID_BUBBLE` are old names that should not be exposed in the `alfasim-sdk`. Couldn't you add these hard-coded constants directly in the `alfasim_plugin_hydrates`?

It seems that the general user **should not** have access to `FLUID_DROPLET` and `FLUID_BUBBLE`

Reply:
Since the plugin developer would use directly the strings `"droplet"` or `"bubble"`, I think it is nice to create a tag and make it available to everyone. 
Note that it could be used to change the layers and phases by a plugin in order to create a specific MultiFieldDescription. 

Do you think it could be useful?

Reply:
"water in oil" is a bubble or droplet? and what about "oil in water"?

And what if a plugin add a new phase? How does the new phase interact (declare how interact?) with other phases to create droplets and bubbles?

Reply:
> "water in oil" is a bubble or droplet?

None of them.

> and what about "oil in water"?

In this situation, it is not needed.

> And what if a plugin add a new phase? How does the new phase interact (declare how interact?) with other phases to create droplets and bubbles?

A plugin can update a phase or a layer by adding a new field to it. And if the MFD used by the user has no bubble or droplet, it can be added by the plugin to have the complete MFD needed to support its models.

We make something similar with the water phase in the hydrates plugin. We ensure the user do not choose a MFD with water and the plugin adds a new phase of water as a dispersed one updating the oil layer (different from our own triphase MFD which has free water, or layer water)

Reply:
The main issue here is that we have just down below

```python
FLUID_PHASE_NAMES = [FLUID_GAS, FLUID_OIL, FLUID_WATER]
```


and now if I'm a new guy trying to understand the sdk I see

```python
FLUID_GAS = "gas"
FLUID_OIL = "oil"
FLUID_WATER = "water"
FLUID_DROPLET = "droplet"
FLUID_BUBBLE = "bubble"
```

what are these `FLUID_DROPLET ` and `FLUID_BUBBLE `? phases? fields?

@prusse-martin's questions are relevant. If he is already confused about it, imagine someone completely new. `FLUID_BUBBLE` today is meant to be used by a specific subset of Multi Field Descriptions (e.g., four-phase). I'd rather keep it hard coded in the plugin side for now and add a more consistent solution (that servers for all scenarios, two/three/N-phases) once we've got one.

Reply:
I suggest to remove `FLUID_DROPLET` and `FLUID_BUBBLE`

Reply:
OK. I understand your concern.
I wasn't so worried about it. But you have a point.

- [ ] I will move it to the plugin and change it to have the `FIELD_` prefix instead of `FLUID_`.

Reply:
I was thinking that `*_DROPLET` was an alias to "water in air" and `*_BUBBLE` an alias to "air in water".